pages/radio-group/permutations.page.tsx

```diff
@@ -92,6 +92,17 @@ const permutations = createPermutations<RadioGroupProps>([
       ],
     ],
   },
+  {
+    readOnly: [true],
+    value: ['first'],
+    items: [
+      [
+        { value: 'first', label: 'First Button' },
+        { value: 'second', label: 'Second Button' },
+        { value: 'third', label: 'Third Button', disabled: true },
```

Comment:
Do we need a permutation when a `disabled` radio is selected while in a readOnly state? Maybe an edge case or maybe not even a valid state?

Reply:
I don't think we need it. The example in permutation page represents a state with `readOnly` selected radio, which is, in our case, pretty much the same as `disabled`, because both of them could not be changed.

This behavior is described in [general guidelines](https://cloudscape.design/components/radio-group?tabId=usage): `If one item is selected and inactive, always deactivate the other items in the radio group.`

Reply:
Its maybe why I consider it an edge case, what happens if the styling for disabled is changed? Do we expect to see the disable or readonly styles while it is in readonly mode?

Reply:
when both `disabled` and `readOnly` are true, `disabled` takes precedence